src/cool_seq_tool/mappers/exon_genomic_coords.py

```diff
@@ -521,60 +660,60 @@ async def _get_alt_ac_start_and_end(
                     return None, error
         return tuple(alt_ac_data_values), None
 
-    async def _genomic_to_transcript_exon_coordinate(
+    async def _genomic_to_tx_segment(
         self,
         genomic_pos: int,
         chromosome: str | None = None,
-        alt_ac: str | None = None,
+        genomic_ac: str | None = None,
         transcript: str | None = None,
         gene: str | None = None,
         get_nearest_transcript_junction: bool = False,
         is_start: bool = True,
-    ) -> TranscriptExonDataResponse:
-        """Convert individual genomic data to transcript data
+    ) -> _GenomicTxSeg:
+        """Given genomic data, generate a boundary for a transcript segment.
 
         :param genomic_pos: Genomic position where the transcript segment starts or ends
             (inter-residue based)
         :param chromosome: Chromosome. Must give chromosome without a prefix
-            (i.e. ``1`` or ``X``). If not provided, must provide ``alt_ac``.
-            If ``alt_ac`` is also provided, ``alt_ac`` will be used.
-        :param alt_ac: Genomic accession (i.e. ``NC_000001.11``). If not provided,
-            must provide ``chromosome. If ``chromosome`` is also provided, ``alt_ac``
+            (i.e. ``1`` or ``X``). If not provided, must provide ``genomic_ac``. If
```

Comment:
I'm realizing that we have put like eleventy billion punctuation marks into this docstring and it will never matter because it doesn't get added to the docs